src/CommunityToolkit.Maui/Alerts/Snackbar/Snackbar.shared.cs

```diff
@@ -49,14 +51,18 @@ public string ActionButtonText
 	/// <inheritdoc/>
 	public IView? Anchor { get; init; }
 
-	/// <inheritdoc/>
+	/// <summary>
+	/// Event occurred when <see cref="IsShown"/> changes.
```

Comment:
```suggestion
	/// Occurs when <see cref="IsShown"/> changes.
```
I would just go with "Occurs" people can already see that it's an event